lib/miq_expression/tag.rb

```diff
@@ -19,6 +19,7 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
+    @managed = managed
     @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
```

Comment:
Sorry :blush: - I'd expect that `namespace` would return simply `MANAGED_NAMESPACE` or `USER_NAMESPACE` as the names suggest - currently it's doing way more than that which means you can't reuse `namespace` in your method below, and you have to grab the `managed` value to do the same logic with it again. I'd suggest we either:

a) Change `#namespace` to return the unmodified namespace, and give the thing it's currently doing a new name, or 
b) Give the unmodified namespace a name (base_namespace, or similar) that both `namespace` and your new method can delegate to

Given the choice, I'd prefer a over b even though it may be more work ( :grinning: ) because I think having a thing called `namespace` and things called `MANAGED_NAMESPACE` and `USER_NAMESPACE` being different concepts sort of introduces some cognitive dissonance for me. It seems like there's a missing concept that hasn't been named!